src/components/DetalheItem/DetalheItem.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
```

Comment:
Quando a página do detalhe começa a carregar mas ainda não deu tempo de pegar os dados dos itens, ela fica com uns dados muito doidos (NaN, imagem quebrada...). Isso dá uma impressão ruim pro usuário, ele vai pensar que o site está quebrado quando na verdade, só está carregando. Seria legal dizer isso pro usuário de alguma forma! Seja colocando um loading ou algum tipo de mensagem ao invés de colocar o componente sem os dados

Pra isso, bastaria fazer uma verificação usando um ternário, por exemplo

se o this.props.itemSelecionado está com os dados corretos, mostra o componente. Se não, mostra um texto escrito Carregando... (por exemplo)

![Captura de Tela 2020-11-10 às 14 19 48](https://user-images.githubusercontent.com/41638060/98708519-086b1480-2360-11eb-903f-f1bb8254dd01.png)